selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java

```diff
@@ -168,7 +168,7 @@ public void shouldDebugWebPhpScriptFromNonDefaultPortAndNotFromFirstLine() throw
             NON_DEFAULT_DEBUG_PORT));
 
     projectExplorer.openItemByPath(PATH_TO_LIB_PHP);
-    editor.setBreakPointAndWaitActiveState(14);
+    editor.setBreakpoint(14);
```

Comment:
It's important part of test to ensure that breakpoint becomes an active in some time.